test/test_env.py

```diff
@@ -0,0 +1,15 @@
+"""Contains tests for creating and initializing custom Julia projects"""
+
+import unittest
+from pysr import julia_helpers
+
+
+class TestJuliaProject(unittest.TestCase):
+    def test_custom_shared_env(self):
+        """Test that we can use PySR in a custom shared env"""
+        test_env_name = "@pysr_test_env"
+        julia_helpers.install(julia_project=test_env_name, quiet=True)
+        Main = julia_helpers.init_julia(julia_project=test_env_name)
+        Main.eval("using SymbolicRegression")
+        # TODO: Test that we are actually in the correct env.
```

Comment:
@mkitti what is the best way to test that we are in a shared env here, for all operating systems/package managers?

Reply:
We could check of the absolute path of the environment starts with one of `Pkg.envdir.(DEPOT_PATH)`

Reply:
What is the most reliable way to get the absolute path of the currently-activated environment?

Reply:
`dirname(Base.active_project())`

Reply:
Thanks!